_search-plugins/ltr/logging-features.md

```diff
@@ -0,0 +1,418 @@
+{: .note}
+
+When logging is performed, the new feature is included in the output, as shown in the following example:
+
+``` json
+{
+    "log_entry1": [
+        {
+            "name": "title_query",
+            "value": 9.510193
+        },
+        {
+            "name": "body_query",
+            "value": 10.7808075
+        },
+        {
+            "name": "user_rating",
+            "value": 7.8
+        }
+    ]
+}
+```
+{% include copy-curl.html %}
+
+### Logging values for a proposed feature set
+
+You can create a completely new feature set for experimental purposes, for example, `other_movie_features`, as shown in the following example request:
+
+```json
+PUT _ltr/_featureset/other_movie_features
+{
+    "name": "other_movie_features",
+    "features": [
+        {
+            "name": "cast_query",
+            "params": [
+                "keywords"
+            ],
+            "template": {
+                "match": {
+                    "cast.name": "{% raw %}{{keywords}}{% endraw %}"
+                }
+            }
+        },
+        {
+            "name": "genre_query",
+            "params": [
+                "keywords"
+            ],
+            "template": {
+                "match": {
+                    "genres.name": "{% raw %}{{keywords}}{% endraw %}"
+                }
+            }
+        }
+    ]
+}
+```
+{% include copy-curl.html %}
+
+The feature set, `other_movie_features`, can be logged alongside the live production set, `more_movie_features`, by appending it as another filter, as shown in the following example request:
+
+```json
+POST tmdb/_search
+{
+"query": {
+    "bool": {
+        "filter": [
+            { "sltr": {
+                "_name": "logged_featureset",
+                "featureset": "other_movie_features",
+                "params": {
+                    "keywords": "rambo"
+                }
+    }},
+            {"match": {
+                "_all": "rambo"
+            }}
+        ]
+    }
+},
+"rescore": {
+    "query": {
+        "rescore_query": {
+            "sltr": {
+                "params": {
+                    "keywords": "rambo"
+                },
+                "model": "my_model"
+            }
+        }
+    }
+}
+}
+```
+{% include copy-curl.html %}
+
+You can continue adding as many feature sets as needed for logging.
+
+## Logging scenarios
+
+Once you have covered the basics, you can consider some real-life feature logging scenarios.
+
+First, logging is used to develop judgment lists from user analytics to capture the exact value of a feature at the precise time of interaction. For instance, you may want to know the recency, title score, and other values at the precise time of a user's interaction. This would help you analyze what correlated with relevance while training. To achieve this, you can build a comprehensive feature set for future experimentation. 
```

Comment:
```suggestion
First, logging is used to develop judgment lists from user analytics to capture the exact value of a feature at the precise time of interaction. For instance, you may want to know the recency, title score, and other values at the precise time of a user's interaction. This would help you analyze which features or factors had relevance while training. To achieve this, you can build a comprehensive feature set for future experimentation. 
```